pages/api/lastPlayed/index.js

```diff
@@ -0,0 +1,28 @@
+import dbConnect from "@/db/connect";
+import User from "@/db/models/User";
+import { getToken } from "next-auth/jwt";
+
+export default async function handler(req, res) {
```

Comment:
please use more descriptive names for your parameters, such as `request` and `response`